packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -532,10 +532,11 @@ class DropdownButtonHideUnderline extends InheritedWidget {
 ///
 /// {@tool snippet --template=stateful_widget_scaffold}
 ///
-/// This sample shows a `DropdownButton` with a customized icon, text style,
-/// and underline and whose value is one of "One", "Two", "Free", or "Four".
+/// This sample shows a `DropdownButton` with a large arrow icon,
+/// purple text style, and bold purple underline, whose value is one of "One",
+/// "Two", "Free", or "Four".
 ///
-/// ![A screenshot of the dropdown button](https://flutter.github.io/assets-for-api-docs/assets/material/dropdown_button.png)
+/// ![](https://flutter.github.io/assets-for-api-docs/assets/material/dropdown_button.png)
```

Comment:
I don't know that leaving this alt empty is better than a poor description. I did an audit of alt text in the codebase though, and found many empty alt texts. 

According to  https://html.spec.whatwg.org/multipage/images.html#alt it should not be empty though. It also states that it should not repeat information that has already been provided. 

@Hixie, do you have any thoughts?

Reply:
I am inclined to think it is okay since it seems to be the prevailing pattern in the codebase.  
If not, we can always return and update. 🙂